src/internet_identity/tests/integration/upgrade.rs

```diff
@@ -315,3 +317,105 @@ fn upgrade_and_rollback_keeps_accounts_intact() {
     assert_eq!(accounts_before, accounts_between);
     assert_eq!(accounts_between, accounts_after);
 }
+
+// TODO: Remove this test after the migration takes place.
+#[test]
+fn upgrade_and_rollback_with_realistic_data_migration() {
```

Comment:
20 doesn't see "realistic" data, no? We have 2M anchors...

Reply:
We can't have it realistic, since a manual run with 2020 anchors takes about 10 min already. 

I think the fact that we're batching the migration means it's almost the same if we have 2 batches or 1000 batches.